src/nav/infrastructure.yml

```diff
@@ -181,6 +179,8 @@ pages:
         pages:
           - title: Prometheus integrations
             path: /docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro
+          - title: Apache Flink integration 
```

Comment:
Are you sure that this new page goes here? Inside the Prometheus integrations list? I've talked with Mikel Angulo and he doesn't know anything about these new 2 integrations for Prometheus.